perfkitbenchmarker/gcp/gce_virtual_machine.py

```diff
@@ -47,8 +48,8 @@
 FLAGS = flags.FLAGS
 
 SET_INTERRUPTS_SH = 'set-interrupts.sh'
-BOOT_DISK_SIZE_GB = 10
-BOOT_DISK_TYPE = disk.STANDARD
+BOOT_DISK_SIZE_GB = 50
+BOOT_DISK_TYPE = disk.REMOTE_SSD
```

Comment:
Is it ok to just change the defaults as part of this CL? If it could affect benchmark performance, seems preferable to do this separately.